doc/external_resources.md

```diff
@@ -10,14 +10,91 @@ to submit a pull request with your recommended addition to the
 
 ## Tutorials
 
-- [2022 EGU SC5.2: Crafting beautiful maps with PyGMT](https://github.com/GenericMappingTools/egu22pygmt) by Wei Ji Leong, Leonardo Uieda, Max Jones, Andre Belem
-- [2021 PyGMT course at the UAF Geophysical Institute](https://github.com/liamtoney/gi-pygmt-2021) by Liam Toney
-- [2021 Remote Online Sessions for Emerging Seismologists (ROSES): Unit 5 - Mapping](https://www.youtube.com/watch?v=Zvcy7VDuhiw) by Liam Toney
-- [2020 Remote Online Sessions for Emerging Seismologists (ROSES): Unit 8 - PyGMT](https://www.iris.edu/hq/inclass/lesson/728) by Liam Toney
-- [PyGMT Tutorial in 2021](https://github.com/MIGG-NTU/PyGMT2021) by MIGG-NTU
-- [PyGMT Workshop at FOSS4G Oceania 2019](https://github.com/GenericMappingTools/foss4g2019oceania) by Wei Ji Leong
+:::::{grid} 1 2 2 3
+
+::::{grid-item-card} 2022 EGU SC5.2: Crafting beautiful maps with PyGMT
+:link: https://www.generic-mapping-tools.org/egu22pygmt/
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://user-images.githubusercontent.com/3974108/171653272-efaceeb1-a253-44a1-bddc-36bc9498c19b.png)
++++
+Wei Ji Leong, Leonardo Uieda, Max Jones, Andre Belem
+::::
+
+::::{grid-item-card} 2021 PyGMT course at the UAF Geophysical Institute
+:link: https://github.com/liamtoney/gi-pygmt-2021
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://user-images.githubusercontent.com/3974108/172042115-3f700c16-0f78-49d3-9e8d-87c77e4c0444.png)
++++
+Liam Toney
+::::
+
+::::{grid-item-card} 2021 Remote Online Sessions for Emerging Seismologists (ROSES): Unit 5 - Mapping
+:link: https://www.youtube.com/watch?v=Zvcy7VDuhiw
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://user-images.githubusercontent.com/3974108/172055872-b81057b6-aef3-473f-8f42-1cc82a68d188.jpg)
++++
+Liam Toney
+::::
+
+::::{grid-item-card} 2020 Remote Online Sessions for Emerging Seismologists (ROSES): Unit 8 - PyGMT
+:link: https://www.iris.edu/hq/inclass/lesson/728
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://user-images.githubusercontent.com/3974108/172055922-8df68c00-4115-42f4-83ae-13a971fef846.png)
++++
+Liam Toney
+::::
+
+::::{grid-item-card} PyGMT Tutorial in 2021
+:link: https://github.com/MIGG-NTU/PyGMT2021
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://user-images.githubusercontent.com/3974108/171653272-efaceeb1-a253-44a1-bddc-36bc9498c19b.png)
```

Comment:
Need a proper thumbnail, let me try and run the notebook using https://mybinder.org/v2/gh/MIGG-NTU/PyGMT2021/master and see if I can get one up.

Reply:
Hmm, the figures seem to be about identical to the PyGMT gallery examples. Perhaps we just add a screenshot of the GitHub repo at https://github.com/MIGG-NTU/PyGMT2021?

```suggestion
![](https://user-images.githubusercontent.com/23487320/172058757-9b68f7ae-7a3f-4f48-9df6-9a065d6cd448.png)
```

![image](https://user-images.githubusercontent.com/23487320/172058757-9b68f7ae-7a3f-4f48-9df6-9a065d6cd448.png)